map_claims.go

```diff
@@ -13,7 +13,14 @@ type MapClaims map[string]interface{}
 // Compares the aud claim against cmp.
 // If required is false, this method will return true if the value matches or is unset
 func (m MapClaims) VerifyAudience(cmp string, req bool) bool {
-	aud, _ := m["aud"].(string)
+	aud, _ := m["aud"].([]string)
```

Comment:
I would make it explicit and use `ok` as second value, then you can check if it's `!ok` proceed as string.

Reply:
if this first one fails we need to do the check on --> .(string)

Iv updated the section to match.